scripts/unest_strategy.ts

```diff
@@ -0,0 +1,44 @@
+import 'dotenv/config';
+import db from '../src/helpers/mysql';
+
+const STRATEGY_NAME = 'multichain';
+
+// Usage: yarn ts-node scripts/unest_strategy.ts
+async function main() {
+  const spaces = await db.queryAsync(
+    `SELECT settings->'$.strategies[*].name', id, settings
+    FROM spaces
+    WHERE JSON_CONTAINS(settings->'$.strategies[*].name', ?)
```

Comment:
Not sure if someone is using multichain inside delegation related strategies, you know the query?

Reply:
Yes, we have around 20 spaces using it in innes strategies. This PR will only target top level strategies

Reply:
Dang, whats the query?

Reply:
Not sure, they are the leftover after running the script

Reply:
You see any `multichain` inside `multichain`?

Reply:
No, it's inside other strategier, like `Math`